06-aks-cluster.md

```diff
@@ -12,14 +12,16 @@ Now that your [ACR instance is deployed and ready to support cluster bootstrappi
    GITOPS_REPOURL=$(git config --get remote.origin.url)
    ```
 
+   > :warning: If you are using your own fork, please ensure this repo is reacheable as well as public accessible from your spoke vnet, and check you are assigning its web URL (`HTTPS`).
```

Comment:
does "reachable" mean "public" and does "public acccessible from your spoke vnet" mean reachable?  Seems like maybe they are backwards here?  I'm not sure what customer problem we're trying to solve here -- following the directions the repo could not be private and there already are firewall rules to make sure github is reachable.  What is new about this being in bicep that requires this?

Reply:
we need that to be accessible over HTTP from the spoke and without credentials (for this RI).

Reply:
Right, how does the migration to bicep impact this?  I'm missing why this is a new concern if someone just follows the instructions what issue is now happening that is warranting this extra concern?

Reply:
it was just a lesson learn from latest run and based on my current setup env and of course it ended up not working properly. In my case, I wasn't exactly under the excepted configuration, and also Im envisioning more advanced users that won't relay on a fork button in GH. 

Open to go silent on this, and revert this commit.

Reply:
Yeah, let's revert.  Sounds like this could have been a problem ever since we introduced flux -- but no issues have ever been opened on it.  Keep an scary-looking messages out of the happy path for a very specific edge case sounds good to me.

Reply:
sure thing @ckittel 

done | addressed from 09b875bd7d8810ceec234132b62aa733796e1e4e

Reply:
Reverted.